data/data.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass, field
+from string import ascii_lowercase
+
+from faker import Faker
+
+
+faker = Faker()
+
+
+@dataclass
+class Customer:
+    post_code: int = field(default_factory=lambda: faker.random_number(digits=10))
+    last_name: str = field(default_factory=lambda: faker.last_name())
+
+    def __post_init__(self):
+        post_code_str = str(self.post_code)
+        self.first_name = ''
+
+        while len(post_code_str) > 0:  # формирование имени на основании почтового кода
+            index = int(post_code_str[0:2])
+            letter = ascii_lowercase[index % 26]
```

Comment:
Предлагаю немного другую реализацию, чтобы переменные были более понятные для чтения.
```suggestion
            letter_index = int(post_code_str[0:2]) % 26
            letter = ascii_lowercase[letter_index]
```